pkg/cmd/server/origin/master_config.go

```diff
@@ -359,7 +361,7 @@ func newAuthenticator(config configapi.MasterConfig, restOptionsGetter restoptio
 			publicKeys = append(publicKeys, publicKey)
 		}
 		tokenAuthenticator := serviceaccount.JWTTokenAuthenticator(publicKeys, true, tokenGetter)
```

Comment:
for sanity, name the local vars something specific like `serviceAccountTokenAuthenticator` here, or `oauthTokenAuthenticator`/`oauthTokenRequestAuthenticators` below